apps/sms/services/test/unit/message_manager_test.js

```diff
@@ -412,91 +412,126 @@ suite('message_manager.js >', function() {
   });
 
   suite('markThreadRead()', function() {
-    setup(function() {
-      this.sinon.spy(MockNavigatormozMobileMessage, 'getMessages');
-      this.sinon.spy(MockNavigatormozMobileMessage, 'markMessageRead');
-
-      MessageManager.markThreadRead(1);
-    });
+    var cursor, threadId;
 
-    test('call mark read on the correct messages', function() {
-      assert.ok(
-        MockNavigatormozMobileMessage.getMessages.calledWithMatch({
-          threadId: 1,
-          read: false
-        })
-      );
+    setup(function() {
+      threadId = 1;
 
-      var messagesList = [
+      var testMessages = [
         { id: 1 },
         { id: 2 },
         { id: 3 },
         { id: 4 }
       ];
 
-      MockNavigatormozMobileMessage.mTriggerMessagesRequest(messagesList);
+      var filter = {
+        threadId: threadId,
```

Comment:
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions

Basically:
```javascript
var o = { a: a, b: b };
```
is equivalent to
```javascript
var o = { a, b };
```

(given the variables `a` and `b` exist of course).

So you can simply use `threadId` instead of `threadId: threadId`; it's shorter.  But this is really a nit !